wire-ios/Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift

```diff
@@ -368,13 +366,17 @@ final class ConversationImagesViewController: UIViewController {
     }
 
     private func createNavigationTitle() {
-        guard let sender = currentMessage.senderUser, let serverTimestamp = currentMessage.serverTimestamp else {
+        guard let sender = currentMessage.senderUser,
+              let serverTimestamp = currentMessage.serverTimestamp else {
             return
         }
-        navigationItem.titleView = TwoLineTitleView(first: (sender.name ?? "").localized.attributedString,
-                                                    second: serverTimestamp.formattedDate.attributedString)
-        navigationItem.titleView?.accessibilityTraits = .header
-        navigationItem.titleView?.accessibilityLabel = "\(sender.name ?? ""), \(serverTimestamp.formattedDate)"
+
+        let titleView = TwoLineTitleView(
+            first: (sender.name ?? "").localized.attributedString,
```

Comment:
A user's name should not be passed into `NSLocalizedString`, right? 
Since you're touching this code, could you please fix it?

Reply:
Fixed it!